sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj

```diff
@@ -37,6 +37,7 @@
     <Compile Include="$(AzureCoreSharedSources)PageResponseEnumerator.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)DiagnosticScopeFactory.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)OperationHelpers.cs" LinkBase="Shared" />
+    <Compile Include="$(AzureCoreSharedSources)OperationInternal.cs" LinkBase="Shared" />
```

Comment:
We're planning to GA in a couple weeks and won't have another preview. I'd prefer to hold off on Key Vault for now. You can open an issue and I can do this afterward.

Reply:
Removed KV from this PR. I'll create issues for each concerned client once we merge the PR.